docs/cow-protocol/concepts/product-features/builder-friendly/milkman.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 4
+---
+
+# Milkman orders
+
+Through Milkman, instead of specifying a ‘min out’ to receive, you specify an **on-chain** data source to pull the market price of your trading pair. 
+
+For example, if you’re swapping `ETH` / `USDC`, you can supply the `ETH` / `USDC` Chainlink price feed. Milkman will automatically adjust the minimum price that you should receive, giving you better on-chain price guarantees and reducing the chances of placing orders with stale prices.
+
+Let’s say you’re looking to sell 1,000 `ETH` and you want to make sure to get the most accurate price for your trade, even if it’s far out in the future. By sending your order through Milkman, you can rely on a price oracle to guarantee that you receive a price that is at least 99.5% (or another custom percentage) of the oracle price at any moment in time. 
```

Comment:
Maybe add an example of what problem this solves (better surplus protection than EBBO for large orders)